kiwi/models/utils.py

```diff
@@ -213,6 +220,28 @@ def make_loss_weights(nb_classes, target_idx, weight):
     return weights
 
 
+def load_torch_file(file_path):
+    file_path = Path(file_path)
+    if not file_path.exists():
+        logger.error('Torch file not found: {}'.format(file_path))
+        return None
```

Comment:
What about throwing an exception?

Reply:
I'm not throwing an exception here as this is a utility used by other functions throughout the code base. As such, these errors should be handled where appropriate. `trainer.resume` for example tests the return and if it is `None` it will try alternate approaches until none works and only then it raises the exception.

Although you might argue that it would be better to raise an exception here and catch it some place else. I am not sure between these two approaches.

Reply:
@kepler might have better feedback on this

Reply:
In this specific case, this function is used by the trainer and model classes and I've implemented exceptions in both. Since they have slightly different use cases I think it might be worth keeping this type of error handling. 
(instead of just raising the same exception in both cases)

Reply:
Agree. Raising an exception in this case is the "least unexpected" behaviour.